app/src/main/java/com/sun/spoonacular/utils/ImageExt.kt

```diff
@@ -0,0 +1,11 @@
+package com.sun.spoonacular.utils
+
+import android.content.Context
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+
+fun ImageView.loadUrl(context: Context, imageApi: String) {
```

Comment:
mỗi view đều chứa context cho riêng nó nên mình sẽ lấy context ở ImageView luôn 
```suggestion
fun ImageView.loadUrl(url: String) {
```